docs/features/api_authentication.md

```diff
@@ -0,0 +1,21 @@
+# API Authentication
```

Comment:
For API authentication with API key, we usually do the authentication only with the Bearer header, and we would include it in all the requests. We don't have an authentication endpoint for APIs.
Why did you decide to use session? Maybe it was discussed somewhere else and I'm just not aware

Reply:
Sorry for the late reply, I came to this conclusion because I didn't really know how to make the difference between a request coming from a regular user and an API key bearer because they both eventually end up making a GET request to access the website. I'll think of another way to recognize an API call to make it more conventional.

Reply:
We should rework the way API is functionning. At first, there were a will of having the same link for API and normal pages. But, actually, it would be easier to have special path for api (/api/...).